src/plugins/d3/renderer/hooks/useSeries/utils.ts

```diff
@@ -1,8 +1,11 @@
 import memoize from 'lodash/memoize';
 import {PreparedLegendSymbol, PreparedSeries, StackedSeries} from './types';
-import {ChartKitWidgetSeries, RectLegendSymbolOptions} from '../../../../../types';
-import {DEFAULT_LEGEND_SYMBOL_PADDING, DEFAULT_LEGEND_SYMBOL_SIZE} from './constants';
+import {ChartKitWidgetSeries} from '../../../../../types';
 import {getRandomCKId} from '../../../../../utils';
+import {DEFAULT_LEGEND_SYMBOL_PADDING} from './constants';
+import {SymbolType} from '../../../../../constants';
+
+const DEFAULT_LEGEND_SYMBOL_SIZE = 8;
```

Comment:
Why is the constant `DEFAULT_LEGEND_SYMBOL_SIZE` declared separately here? It seems to completely duplicate the name and value of the constant from `'./constants'`

Reply:
One is for legend, second one is for series. But i agree that this is unclear.

Reply:
DEFAULT_LEGEND_SYMBOL_SIZE always used for a legend, is it not?